internal/cltest/cltest.go

```diff
@@ -564,3 +564,21 @@ func GetAccountAddress(store *store.Store) common.Address {
 func StringToHash(s string) common.Hash {
 	return common.BytesToHash([]byte(s))
 }
+
+// AssertServerResponse is used to match against a client response, will print
+// any errors returned if the request fails.
+func AssertServerResponse(t *testing.T, resp *http.Response, expectedStatusCode int) {
+	if resp.StatusCode == expectedStatusCode {
```

Comment:
It'd be nice to consolidate this with [cltest.GetStatusCode](https://github.com/smartcontractkit/chainlink/blob/e2e029e696d6c2723f6667ff0e3601970c14efad/internal/cltest/cltest.go#L301). Yours seems more idiomatic.